tests/pipelines/test_pipelines_common.py

```diff
@@ -64,7 +64,8 @@ def get_tiny_config_from_class(configuration_class):
     camel_case_model_name = configuration_class.__name__.split("Config")[0]
 
     try:
-        module = importlib.import_module(f".test_modeling_{model_type.replace('-', '_')}", package="tests")
+        model_slug = model_type.replace("-", "_")
+        module = importlib.import_module(f".test_modeling_{model_slug}", package=f"tests.{model_slug}")
```

Comment:
BIG MODIFICATION.

NO model tester was tested anymore on the pipelines since the change of directory, this re-eanbles ALL of them

Reply:
That part should go in its own PR, I believe.

Reply:
https://github.com/huggingface/transformers/pull/15924